### DISCUSSION TITLE
r/city-china-zhejiang-ningbo-xiangshan

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#50] [Announcements]
# r/city-china-zhejiang-ningbo-xiangshan

游记｜宁波象山，北纬30度最美海岸线

https://blog.ops-coffee.cn/r/city-china-zhejiang-ningbo-xiangshan

**Comment by blackox626** [comment]
五一 带娃  象山半边山观堂 躺尸三天 😄

**Reply by ops-coffee** [reply]
这个也挺好的，简单又放松，不过半边天人实在是太多了

**Reply by blackox626** [reply]
没去别的地方 没法对比
不过问酒店的服务生 说明显没有去年人多
从我个人判断，停车场有空位，顺便还能冲个电
景区附近 吃饭也不需要等位置

**Comment by zhangxt2015** [comment]
礼貌询问房车如何解决洗漱问题

**Reply by ops-coffee** [reply]
车上有水箱，有淋浴间，洗漱会麻烦一点，不过也不是不可以

**Comment by creazy412** [comment]
慕了，感觉缺少点房车内饰的照片，想看看多大的空间，以备也想整一台了

**Reply by ops-coffee** [reply]
关于房车可以看这里：https://blog.ops-coffee.cn/r/index.html
